super_gui/src/components/DoorManUI.tsx

```diff
@@ -0,0 +1,166 @@
+                    </label>
+                    <select
+                      className="border border-beige rounded p-2"
+                      value={selectedLocker}
+                      onChange={(e) => setSelectedLocker(e.target.value)}
+                    >
+                      <option value="">Select a locker</option>
+                      <option value="1">Locker 1</option>
+                      <option value="2">Locker 2</option>
+                      <option value="3">Locker 3</option>
+                      <option value="4">Locker 4</option>
+                      <option value="5">Locker 5</option>
+                      <option value="6">Locker 6</option>
+                      <option value="7">Locker 7</option>
+                    </select>
+                  </div>
+                  <button
+                    className="bg-accentBlue text-white py-2 px-4 rounded font-body w-full"
+                    onClick={assignPackage}
+                  >
+                    Submit
+                  </button>
+                </div>
+
+                <p className="mt-auto text-charcoal italic text-center">
+                  Packages will be removed after 7 days if not picked up.
+                </p>
+              </div>
+            </div>
+
+            {/* Locker Status Section - RIGHT */}
+            <div className="md:w-1/2 flex justify-center">
+              <div className="flex flex-col items-center justify-start border rounded-2xl p-6 w-full max-w-sm shadow-sm hover:shadow transition bg-white h-full">
+                <div className="w-full mb-6">
+                  <h2 className="text-xl text-center font-medium font-[Roboto Condensed]">
+                    Locker Status
+                  </h2>
+                  <hr className="border-[#D3C9B8] w-full mt-2" />
+                </div>
+
+                <div className="w-full">
+                  <table className="w-full">
+                    <thead className="sticky top-0 bg-white">
+                      <tr className="border-b border-beige">
+                        <th className="p-2 text-left font-body border-b border-beige">
+                          Locker Number
+                        </th>
+                        <th className="p-2 text-center font-body border-b border-beige">
+                          Status
+                        </th>
+                        <th className="p-2"></th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      {lockers.map((locker) => (
+                        <tr key={locker.id} className="border-b border-beige">
+                          <td className="p-2 text-left border-r border-beige">
+                            {locker.lockerNumber}
+                          </td>
+                          <td
+                            className={`p-2 text-center border-r border-beige ${
+                              locker.status === "Empty"
```

Comment:
Locker.status is not  a field on the locker. you have apartmentNumber in the object. When apartment number is null, it is empty. If it is a number, it is full.